Server/Plugins/APIDump/APIDesc.lua

```diff
@@ -16997,6 +17010,21 @@ end
 				{
 					Include = "^E_META_.*",
 				},
+				EffectID = 
```

Comment:
Erm... the other constant group names seem to just be the name of whatever the enum is. Can I have a hint?